package.json

```diff
@@ -2,7 +2,7 @@
   "name": "clinic",
   "description": "Clinic.js diagnoses your performance issues",
   "repository": "nearform/node-clinic",
-  "version": "7.0.0",
+  "version": "8.0.0",
```

Comment:
Please revert this, we will bump the version on release.

Reply:
O ok, done.